.github/workflows/testAndPublish.yml

```diff
@@ -0,0 +1,243 @@
+name: Run tests and publish
+
+on:
+  push:
+    branches:
+    - master
+    - beta
+    - rc
+    - 'try-**'
+    tags:
+    - 'release-**'
+
+  pull_request:
+    branches:
+    - master
+    - beta
+    - rc
+
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  START_BUILD_NUMBER: 100000
+  pullRequestNumber: ${{ github.event_name == 'pull_request' && github.event.number }}
+  scons_publisher: NV Access
+  # Comment out any of the feature_* variables to disable the respective build feature.
+  # They are checked for existence of content, not specific value.
+  # feature_buildSymbols: configured
+  # feature_uploadSymbolsToMozilla: configured
+  # feature_buildAppx: configured
+  # feature_crowdinSync: configured
+  # feature_signing: configured
+
+defaults:
+  run:
+    shell: cmd
```

Comment:
Wouldn't it be better to keep pwsh as default shell, since there are many instances now where powershell is called explicitly.

Reply:
@nvdaes thoughts on this? I thought this design was intentional

Reply:
Yes, it was intentional since I was not successful trying to use powershell as the default shell. I'll try in a testing branch using again powershell, and also psh.